infrastructure/env/production/tfvars.json

```diff
@@ -3,26 +3,10 @@
   "environment_short": "prod",
   "location": "West Europe",
   "cosmosdb_failover_location": "North Europe",
-  "azurerm_resource_group": "agid-rg-prod",
```

Comment:
i feat that some typescript task use this values (see lib/config.ts). have you checked that it's not the case ?

Reply:
I guess [this list](https://github.com/teamdigitale/digital-citizenship/blob/master/lib/config.ts#L59) needs to be reviewed as we moved many tasks to TF provisioners?

Reply:
for example the api_management task read all the values from tsconfig.json to replace them in the API management templates

Reply:
we can close this PR in favor of #54 which resolves the issue

Reply:
great, thanks